pkg/acsengine/defaults.go

```diff
@@ -80,6 +80,17 @@ func setOrchestratorDefaults(cs *api.ContainerService) {
 		if a.OrchestratorProfile.KubernetesConfig.NetworkPolicy == "" {
 			a.OrchestratorProfile.KubernetesConfig.NetworkPolicy = DefaultNetworkPolicy
 		}
+
+		if a.OrchestratorProfile.KubernetesConfig.DnsServiceIP == "" {
+			a.OrchestratorProfile.KubernetesConfig.DnsServiceIP = DefaultKubernetsDnsServiceIP
+		}
+		if a.OrchestratorProfile.KubernetesConfig.ServiceCIDR == "" {
+			a.OrchestratorProfile.KubernetesConfig.ServiceCIDR = DefaultKubernetsServiceCIDR
```

Comment:
Can you validate that the DnsServiceIP falls into the ServiceCIDR? (and isn't the .255 since I think you got tripped up on that)